src/encoded/schemas/human_donor.json

```diff
@@ -121,64 +153,86 @@
         },
         "parents": {
             "title": "Parents",
-            "description": "Donor IDs of biological parents, if known.",
+            "description": "Biological parents of this donor",
+            "comment":"This is a link for cases when families are all donors in our system. Children will be calculated from this link.",
             "type":  "array",
             "default": [],
             "uniqueItems": true,
             "items": {
                 "title": "Parent",
-                "description": "A donor ID of a biological parent, if known.",
-                "comment": "For human biosamples, see human_donor.json for available identifiers. For mouse biosamples, see mouse_donor.json for available identifiers.",
+                "description": "Biological parent of this donor.",
+                "comment": "For human biosamples, see human_donor.json for available identifiers.",
                 "type": "string",
                 "linkTo": "HumanDonor"
             }
         },
         "children": {
```

Comment:
I think this should be removed from schema.

Reply:
removed children